### PR TITLE
add project prefix to the artifact name

### PR DESCRIPTION
**Pull request by avichay77**
version 1.0.5
add optional project prefix to artifact name

### CODE REVIEW

File: maven-uploader-plugin/pom.xml [hunk 1]
```diff
@@ -120,6 +120,9 @@
             <plugin>
               <groupId>org.apache.maven.plugins</groupId>
               <artifactId>maven-gpg-plugin</artifactId>
+                <configuration>
```
Comment (drorweiss) [inline maven-uploader-plugin/pom.xml]: I guess you didn't mean to commit skip gpg?

File: README.md [hunk 2]
```diff
@@ -9,11 +9,11 @@ To submit your code follow these simple steps:
 * Build your project including sources and send to Codota for indexing by running the following command in your project directory
     
     ```
-    mvn clean source:jar source:test-jar jar:test-jar package com.codota:codota-uploader:1.0.4:upload -Dcodota.token={your_token_here} -Dcodota.endpoint=https://upload.codota.com/artifacts -DskipAssembly -DskipTests -Dmaven.javadoc.skip=true -Dgpg.skip
+    mvn clean source:jar source:test-jar jar:test-jar package com.codota:codota-uploader:1.0.5:upload -Dcodota.token={your_token_here} -Dcodota.endpoint=https://upload.codota.com/artifacts -DskipAssembly -DskipTests -Dmaven.javadoc.skip=true -Dgpg.skip
     ```
     Also, you have the following optional parameters:
     ```
-    -Dcodota.repoName={your_repo_name} -Dcodota.lastPushed={utc_format_datetime}
+    -Dcodota.repoName={your_repo_name} -Dcodota.lastPushed={utc_format_datetime} -Dcodota.projectPrefix={prefix}
```
Comment (drorweiss) [inline README.md]: Not related to your change, but I think repoName and lastPushed belong only in our internal version of this plugin for public code and don't here

**Review by drorweiss — APPROVED** [review]
LGTM, just see the comments below.
Also, seems the addition of groupId and artifactId as separate query params to the upload was lost in the recent revert. Probably need to bring it back surgically.

File: maven-uploader-plugin/pom.xml (hunk 1 above)
Comment (avichay77) [reply maven-uploader-plugin/pom.xml]: right. my mistake.

File: README.md (hunk 2 above)
Comment (avichay77) [reply README.md]: I'll remove it from README, but it's still supported.

**Pull request merged**